src/test/resources/test_data/seed-licence-summaries.sql

```diff
@@ -1,9 +1,10 @@
 insert into licence (
   id,
+  version,
   com_staff_id,
   type_code,
   status_code
 ) values
-(1,1,'AP','IN_PROGRESS'),
-(2,1,'AP','APPROVED'),
-(3,1,'AP','REJECTED');
+(1,'1.0',1,'AP','IN_PROGRESS'),
+(2,'1.0',1,'AP','APPROVED'),
+(3,'1.0'',1,'AP','REJECTED');
```

Comment:
Is the second apostrophe in `'1.0''` intentional?

Reply:
this wasn't being used. deleted the file